nf-core-contributors.yaml

```diff
@@ -21,6 +21,7 @@ contributors:
       contact: Phil Ewels
       contact_email: phil.ewels@scilifelab.se
       contact_github: ewels
+      location: [59.3505174, 18.0221508]
```

Comment:
Question: is it possible to locate the pins based on an address search, instead of having to specify coordinates? Minor convenience thing only, so only worth it if it can be done easily.

Reply:
After some time, I've found a way how to do this.